.github/workflows/test.yaml

```diff
@@ -37,3 +37,16 @@ jobs:
         with:
           version: v1.55
           args: --timeout=5m
+
+  asset-test-build:
+    name: test build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: "go.mod"
+          go-version: "1.21"
```

Comment:
> If both the go-version and the go-version-file inputs are provided then the go-version input is used.
https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file

you can remove `go-version`.

Reply:
fixed